.github/workflows/flake8.yml

```diff
@@ -0,0 +1,24 @@
+name: Flake8 Check
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  flake8:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
```

Comment:
I don't think is a good idea to let that version so wide, check if the action can take the version from pyproject.tml, so we don't have to write that number in various places.

And this reminded me that we chose a target version kind of old (https://github.com/PEM-Humboldt/stac-data-tools/blob/main/pyproject.toml#L3). Change that and this one to a newer one (3.11 or 3.12) (https://devguide.python.org/versions/)

Reply:
Done